app/views/users/vimeo_search.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="col-sm-6">
+  <% if @vimeo_results == nil %>
+  <% else %>
+  <h1> Search Results </h1>
+    <% @vimeo_results.each do |video| %>
+    <% photo = video["pictures"] %>
+    <% videos = video["metadata"]["connections"]["videos"]["total"] %>
```

Comment:
The photo and videos variable should be indented under the loop within which they are defined.